health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/SearchFileIds.js

```
@@ -0,0 +1,73 @@
+    const response = await Digit.CustomService.getResponse({
+      url: "/project-factory/v1/data/_search",
+      useCache: false,
+      method: "POST",
+      userService: false,
+      body:body,
+    });
+    // debugger;
+    if (response?.ResourceDetails?.length === 0) {
+      throw new Error("File not found for given microplan");
+    }
+    let fileIds = []
```

Biome

[error] This let declares a variable that is only assigned once.

'fileIds' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)